pkg/nb/types.go

```diff
@@ -391,6 +395,14 @@ type AccountAllowedBuckets struct {
 	PermissionList []string `json:"permission_list"`
 }
 
+// AccountNsfsConfig is the configuration of NSFS of CreateAccountParams
```

Comment:
Hello @liranmauda. The change looks good 😊 It might be a good idea to dedup `type AccountNsfsConfig struct {}` declaration here and also at [pkg/apis/noobaa/v1alpha1/noobaaaccount_types.go](https://github.com/liranmauda/noobaa-operator/blob/ab2e3dba85f35c8c2c54138fd7628b4db9601169/pkg/apis/noobaa/v1alpha1/noobaaaccount_types.go#L80-L86). 

[This](https://github.com/baum/noobaa-operator/commit/5cf5d24272d117b936a9fa8f769d5c361bd0e7e8) might be a way to achieve that.

Reply:
Thanks.
Done.